alf/trainers/policy_trainer.py

```diff
@@ -455,11 +448,6 @@ def play(root_dir,
     root_dir = os.path.expanduser(root_dir)
     train_dir = os.path.join(root_dir, 'train')
 
-    if random_seed is not None:
-        random.seed(random_seed)
-        np.random.seed(random_seed)
-        tf.random.set_seed(random_seed)
-
```

Comment:
Don't forget to common.set_random_seed here?

Reply:
This has been moved to play.py because env is initialized before this.